src/components/Box/__stories__/Box.stories.mdx

```diff
@@ -0,0 +1,491 @@
+import Box from "../Box";
+import {ArgsTable, Story, Canvas, Meta} from "@storybook/addon-docs";
+import {createStoryMetaSettings} from "../../../storybook/functions/create-component-story";
+import styles from "./Box.stories.module.scss";
+
+export const metaSettings = createStoryMetaSettings({
+    component: Box,
+    enumPropNamesArray: [
+        "children",
+        "disabled",
+        "border",
+        "borderTop",
+        "borderEnd",
+        "borderBottom",
+        "borderStart",
+        "borderColor",
+        "rounded",
+        "roundedTop",
+        "roundedEnd",
+        "roundedBottom",
+        "roundedStart",
+        "shadow",
+        "margin",
+        "marginX",
+        "marginY",
+        "marginTop",
+        "marginEnd",
+        "marginStart",
+        "padding",
+        "paddingX",
+        "paddingY",
+        "paddingTop",
+        "paddingEnd",
+        "paddingBottom",
+        "paddingStart",
+        "backgroundColor",
+        "text",
+    ],
+});
+
+<Meta
+    title="Layout/Box/Box"
+    component={Box}
+    argTypes={metaSettings.argTypes}
+    decorators={metaSettings.decorators}
+/>
+
+<!--- Component template -->
+
+export const boxTemplate = (args) => {
+    return <div className={styles['monday-storybook-box-wrapper']}>
```

Comment:
why using different scss convention for our convention :(

Reply:
it will create confusion because people will see not consistent examples

Reply:
this is consistent with our storybook classes.
let's discuss it, have not problem to change it.